MonMaraicher/Features/Map/FarmerService.swift

```diff
@@ -10,9 +10,9 @@ import Foundation
 final class FarmerService {
 
     // TODO: Write unit tests for this method
-    func loadFarmers() async throws -> Farmers {
+    func loadFarmers(latitude: Double, longitude: Double) async throws -> Farmers {
         do {
-            let endPoint = "https://opendata.agencebio.org/api/gouv/operateurs/?activite=Production&filtrerVenteDetail=1&lat=43.600345&lng=3.908215"
+            let endPoint = "https://opendata.agencebio.org/api/gouv/operateurs/?activite=Production&filtrerVenteDetail=1&lat=\(latitude)&lng=\(longitude)"
```

Comment:
I would move that code inside a private func like : `func buildURL(for location: Location) throw -> URL`

Reply:
Check how you can add query parameters in a URL with the native Apple `URL` API.